Building against Mariadb 10.2.6

I am building against MariaDB 10.2.6, which was just released a few days ago as a stable version.  The mysqlclient-python fails to compile with the following error.  To test it's easy to get the docker image:
docker run -it mariadb:10.2.6 bash
apt-get install ...
git clone mysqlclient-python ...
cd ...
python setup.py install

The mariadb:10.2.6 image is based off Debian Jessie 8.8.

I get the following error:
```
> x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security -fPIC -Dversion_info=(1,3,10,'final',0) -D__version__=1.3.10 -I/usr/include/mysql -I/usr/include/python2.7 -c _mysql.c -o build/temp.linux-x86_64-2.7/_mysql.o
> In file included from _mysql.c:30:0:
> /usr/include/mysql/mysql.h:438:3: warning: function declaration isn't a prototype [-Wstrict-prototypes]
>    MYSQL_CLIENT_PLUGIN_HEADER
>    ^
> /usr/include/mysql/mysql.h:583:1: warning: function declaration isn't a prototype [-Wstrict-prototypes]
>  my_bool STDCALL mysql_embedded();
>  ^
> _mysql.c: In function '_mysql_ConnectionObject_ping':
> _mysql.c:1911:41: error: 'MYSQL' has no member named 'reconnect'
>   if ( reconnect != -1 ) self->connection.reconnect = reconnect;
>                                          ^
> error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
> 
```

Thanks for your help!

--- a/_mysql.c
+++ b/_mysql.c
@@ -1908,7 +1908,10 @@ _mysql_ConnectionObject_ping(
 	int r, reconnect = -1;
 	if (!PyArg_ParseTuple(args, "|I", &reconnect)) return NULL;
 	check_connection(self);
-	if ( reconnect != -1 ) self->connection.reconnect = reconnect;
+	if (reconnect != -1) {
+		my_bool recon = reconnect;
+		mysql_options(&self->connection, MYSQL_OPT_RECONNECT, &recon);
+	}
 	Py_BEGIN_ALLOW_THREADS
 	r = mysql_ping(&(self->connection));
 	Py_END_ALLOW_THREADS

